openslides/motions/views.py

```diff
@@ -357,6 +358,23 @@ def check_view_permissions(self):
         return (has_perm(self.request.user, 'motions.can_see') and
                 has_perm(self.request.user, 'motions.can_manage'))
 
+    def update(self, *args, **kwargs):
+        """
+        Customized view endpoint to update a motion poll.
+        """
+        result = super().update(*args, **kwargs)
+        poll = self.get_object()
+        poll.motion.write_log([ugettext_noop('Vote updated')], self.request.user)
+        return result
+
+    def destroy(self, *args, **kwargs):
+        """
+        Customized view endpoint to delete a motion poll.
+        """
+        poll = self.get_object()
+        poll.motion.write_log([ugettext_noop('Vote deleted')], self.request.user)
```

Comment:
Please do not write log first because the destroy call may fail for whatever reason. You can save motion = poll.motion first, then run super to destroy the poll and then run motion.write_log().